src-docs/src/views/toast/toast_list.js

```diff
@@ -62,7 +62,10 @@ export default () => {
             </p>
             <p>
               And some other stuff on another line, just for kicks. And{' '}
-              <OuiLink href="#">here&rsquo;s a link</OuiLink>.
+              <OuiLink href="https://oui.opensearch.org/latest/">
+                here&rsquo;s a link
+              </OuiLink>
+              .
```

Comment:
<img width="600" alt="Screenshot 2023-04-18 at 09 33 05" src="https://user-images.githubusercontent.com/17855243/232664203-c9ce815c-91fa-47c8-b15b-254f7622d299.png">

Reply:
Lets use https://opensearch.org/docs/latest/ for this demo.

Reply:
Made the changes according to the comment

<img width="600" alt="Screenshot 2023-04-19 at 10 12 03" src="https://user-images.githubusercontent.com/17855243/232965199-c7c92d65-2fe3-4a6a-8b8c-cd3373764f4b.png">